### ISSUE TITLE
Advertise extensions that readers answer

### ISSUE THREAD

**Issue by hobu** [PDAL/PDAL#2571]
It would be helpful to have the following output:

```
pdal drivers --showjson
```

```
[
	{
		"description" : "Write data to SQLite3 database files.",
		"link" : "",
		"name" : "writers.sqlite",
                "extensions":["sqlite","sqlite3"]
	},
	{
		"description" : "Text Writer",
		"link" : "http://pdal.io/stages/writers.text.html",
		"name" : "writers.text",
                "extensions":["csv","txt"]
	},
	{
		"description" : "Write data using TileDB.",
		"link" : "http://pdal.io/stages/drivers.tiledb.writer.html",
		"name" : "writers.tiledb"
	}
]
```

**Comment by abellgithub** [comment]
Do you really want the extensions missing if there aren't any?  Seems better to me to have an empty array.

**Comment by hobu** [comment]
> Seems better to me to have an empty array.

I agree.

**Comment by abellgithub** [comment]
Fixed with 105a1a24dd371a6feb41c62a3d1d488040c20056
